src/test/java/uk/gov/companieshouse/extensions/api/attachments/file/FileTransferApiClientUnitTest.java

```diff
@@ -213,10 +206,9 @@ public void testDelete_GenericExceptionResponse() {
 
         when(restTemplate.exchange(eq(DELETE_URL), eq(HttpMethod.DELETE), any(), eq(String.class))).thenThrow(exception);
 
-        expectedException.expect(RestClientException.class);
-        expectedException.expectMessage(exception.getMessage());
-
-        fileTransferApiClient.delete(FILE_ID);
+        RestClientException thrown = assertThrows(RestClientException.class, () ->
+            fileTransferApiClient.delete(FILE_ID));
+        assertTrue(thrown.getMessage().contains(thrown.getMessage()));
```

Comment:
same here for the exception message check (see above)

Reply:
As  2 above